projects/project-player-library/src/lib/project-player-library.module.ts

```diff
@@ -45,14 +45,16 @@ import { MAT_DATE_FORMATS } from '@angular/material/core';
 import { MatDatepickerModule } from '@angular/material/datepicker';
 import {provideNativeDateAdapter} from '@angular/material/core';
 import { FilesCardComponent } from './shared/files-card/files-card.component';
+import { LearningResourcesComponent } from './pages/learning-resources/learning-resources.component';
 
 const routes: Routes = [
   { path: 'details/:id', component: DetailsPageComponent },
   { path: 'files/:id', component: AttachmentListingPageComponent },
   { path: 'task-details/:taskId/:id', component: TaskDetailsPageComponent },
   { path: 'add-task/:id', component: AddTaskPageComponent },
   { path: 'preview-details/:id', component: PreviewDetailsPageComponent},
-  { path: 'add-files/:id', component: AddFilesPageComponent }
+  { path: 'add-files/:id', component: AddFilesPageComponent },
+  { path: 'learning-resource/:taskId/:id/:fromHome', component: LearningResourcesComponent}
```

Comment:
change the fromHome key and do pass it via queryparams, not in the routing path

Reply:
done